src/packages/cli/bin/gw-print-schema.js

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env node
+/* eslint-disable @typescript-eslint/no-var-requires */
+const utils = require('@graphql-tools/utils');
+const { writeFileSync } = require('fs');
+const path = require('path');
+
+const printSchema = async () => {
+	const buildDir = path.join('file://', process.cwd(), `./.graphweaver/backend/index.js`);
+	const { graphweaver } = await import(buildDir);
+
+	if (!graphweaver?.schema) {
+		console.warn(
+			'No schema found. To generate types make sure that you export Graphweaver from your index file.'
+		);
+		process.exit(0);
+	}
+
+	const sdl = utils.printSchemaWithDirectives(graphweaver.schema);
+
+	const args = process.argv;
+
+	if (args.includes('--output') || args.includes('-o')) {
+		const outputIndex = args.indexOf('--output') + 1 || args.indexOf('-o') + 1;
```

Comment:
We should be able to just pass this arg in already parsed and pulled out by `yargs` so we don't have to dig around in `process.argv` ourselves.

Reply:
This script is started as an external bin, so I don't think I can pass it in.

Reply:
Got it, within the docker container.